spock-core/src/main/java/spock/util/concurrent/PollingConditions.java

```diff
@@ -40,6 +40,12 @@
  *   assert machine.efficiency >= 0.9
  * }
  * </pre>
+ *
+ * Warning! Avoiding assert keyword in the clojure is only possible if the conditions object type is known
```

Comment:
It is "closure", not "clojure".
Clojure is a Lisp dialect.

Reply:
My bad, I missed it.

Reply:
That's true, sorry!